geolocator/android/src/main/java/com/baseflow/geolocator/PositionStreamHandlerImpl.java

```diff
@@ -15,8 +15,9 @@
 
 import java.util.Map;
 
-class StreamHandlerImpl implements EventChannel.StreamHandler {
-  private static final String TAG = "StreamHandlerImpl";
+class PositionStreamHandlerImpl implements EventChannel.StreamHandler {
+
+  private static final String TAG = "PositionStreamImpl";
```

Comment:
Rename TAG to match class name:
```suggestion
  private static final String TAG = "PositionStreamHandlerImpl";
```